survey.js

```diff
@@ -4,7 +4,7 @@
 //  surveyman.js is released under the CRAPL.
 
 if (!_) {
-    _ = require("underscore");
+    var _ = require("underscore");
```

Comment:
@etosch , when I was trying to run the tests there is a error I see |_ is undefined| . So, I corrected it.